src/components/Home/Home.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-nested-ternary */
```

Comment:
- [ ] I request that you re-enable the linters and fix them instead. They help to create good quality code that is also well formatted.